spec.emu

```diff
@@ -517,21 +569,79 @@ location: https://github.com/tc39/proposal-decimal/
   </emu-clause>
 
   <emu-clause id="sec-decimal.prototype.tostring">
-    <h1>Decimal128.prototype.toString ( [ _options_ ] )</h1>
+    <h1>Decimal128.prototype.toString ( )</h1>
+    <p>This method performs the following steps when called:</p>
+    <emu-alg>
+      1. Return Decimal128ToDecimalString(*this*).
+    </emu-alg>
+  </emu-clause>
+
+  <emu-clause id="sec-decimal.prototype.toexponential">
+    <h1>Decimal128.prototype.toExponential ( )</h1>
     <p>This method performs the following steps when called:</p>
     <emu-alg>
-      1. Let _roundingMode_ be ? Get(_options_, *"roundingMode"*).
-      1. If _roundingMode_ is *undefined*, set _roundingMode_ to the <emu-xref href="#dfn-decimal128-default-rounding-mode">default rounding mode</emu-xref>.
-      1. If _roundingMode_ is not a <emu-xref href="#dfn-decimal128-rounding-mode">rounding mode</emu-xref>, throw a *RangeError* exception.
-      1. Let _format_ be ? Get(_options_, *"format*").
-      1. If _format_ is *undefined*, set _format_ to *"decimal"*.
-      1. If _format_ is not in « *"decimal"*, *"exponential"* », then throw a *RangeError* exception.
-      1. If _format_ is *"decimal*", then
-        1. Return Decimal128ToDecimalString(*this*).
       1. Return Decimal128ToExponentialString(*this*).
```

Comment:
If this is the only callsite, consider putting the algorithm definition inline (following how Number.prototype.toExponential is specified)

Reply:
We do use the AO in a couple other places. In an earlier version of the spec, with fewer details, this may well have been the one call site for the AO.